projects/storefrontlib/src/cms-structure/services/cms-components.service.spec.ts

```diff
@@ -99,8 +99,8 @@ describe('CmsComponentsService', () => {
       });
     });
     it('should resolve features before emitting values', () => {
-      const featureModulesService = TestBed.inject<MockFeatureModulesService>(
-        FeatureModulesService as any
+      const featureModulesService = TestBed.inject<MockCmsFeaturesService>(
```

Comment:
nitpick: maybe rename the variable here as well

Reply:
should we inject here mocks?

Reply:
We are not injecting mock here, but in the test scenario, we know that it will be a mock and we want to access specific testing methods from it.

Reply:
(and the variable was renamed)